controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStream.java

```diff
@@ -66,7 +66,7 @@
  * This class is coded for transaction ids that follow the scheme that msb 32 bits represent epoch
  */
 class PravegaTablesStream extends PersistentStreamBase {
-    private static final String STREAM_TABLE_PREFIX = "Table" + SEPARATOR + "%s" + SEPARATOR; // stream name
+    private static final String STREAM_TABLE_PREFIX = "Table" + SEPARATOR + "%s" + SEPARATOR + "%s" + SEPARATOR; // stream name
```

Comment:
Another clarification question. I have the same question about this format containing two `%s` and not three. Can you clarify what the format is supposed to be for these constants?

Reply:
It’s the prefix that’s used in next three table name formats